packages/types/src/documents/app/app.ts

```diff
@@ -32,6 +32,8 @@ export interface App extends Document {
   updatedBy?: string
   pwa?: PWAManifest
   scripts?: AppScript[]
+  // stores a list of IDs and when they were last published (timestamp)
+  lastPublishedAt?: Record<string, string>
```

Comment:
This might be really confusing. Will this hold automations and screen changes?

Reply:
Should then be `resourcesPublishedAt`? Otherwise it looks like it is related to the app itself